src/Pages/CreateProduct/index.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
+import { ProductForm } from "../../components/UI/Forms/ProductForm";
 
 export const CreateForm = () => {
```

Comment:
Why the name of the folder is `CreateProduct`  and the name of the component is `CreateForm`?
Please note all these things are super important
```suggestion
export const CreateProduct = () => {
```

Reply:
Fixed